x-pack/plugins/apm/server/lib/observability_overview/get_transaction_per_minute.ts

```diff
@@ -42,23 +47,47 @@ export function getTransactionCoordinates({
             },
           },
           aggs: {
-            distribution: {
-              date_histogram: {
-                field: '@timestamp',
-                fixed_interval: bucketSize,
-                min_doc_count: 0,
+            transactionType: {
+              terms: {
+                field: TRANSACTION_TYPE,
+              },
+              aggs: {
+                timeseries: {
+                  date_histogram: {
+                    field: '@timestamp',
+                    fixed_interval: bucketSize,
+                    min_doc_count: 0,
+                  },
+                },
               },
             },
           },
         },
       });
 
-      return (
-        aggregations?.distribution.buckets.map((bucket) => ({
-          x: bucket.key,
-          y: calculateThroughput({ start, end, value: bucket.doc_count }),
-        })) || []
-      );
+      if (!aggregations || !aggregations.transactionType.buckets) {
+        return { value: undefined, timeseries: [] };
+      }
+
+      const topTransactionTypeBucket =
+        aggregations.transactionType.buckets.find(
+          ({ key: transactionType }) =>
+            transactionType === TRANSACTION_REQUEST ||
+            transactionType === TRANSACTION_PAGE_LOAD
+        ) || aggregations.transactionType.buckets[0];
+
+      return {
+        value: calculateThroughput({
+          start,
+          end,
+          value: topTransactionTypeBucket?.doc_count || 0,
+        }),
+        timeseries:
+          topTransactionTypeBucket?.timeseries.buckets.map((bucket) => ({
+            x: bucket.key,
+            y: calculateThroughput({ start, end, value: bucket.doc_count }),
```

Comment:
Can you use the [rate aggregation](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-metrics-rate-aggregation.html#:~:text=Rate%20aggregationedit,histogram%20fields%20in%20the%20documents.) instead, like here:

https://github.com/elastic/kibana/blob/16a176af98e46a074d939a73c7cca3ef91aa0a39/x-pack/plugins/apm/server/lib/services/get_throughput.ts#L80-L84